packages/ui/components/test-setup.ts

```diff
@@ -1,6 +1,16 @@
 import matchers from "@testing-library/jest-dom/matchers";
 import { cleanup } from "@testing-library/react";
-import { afterEach, expect } from "vitest";
+import { afterEach, expect, vi } from "vitest";
+
+vi.mock("@calcom/lib/OgImages", async () => {
+  return {};
+});
+
+global.ResizeObserver = vi.fn().mockImplementation(() => ({
+  observe: vi.fn(),
+  unobserve: vi.fn(),
+  disconnect: vi.fn(),
```

Comment:
We added it to solve the following error:
![image](https://github.com/calcom/cal.com/assets/121884634/8d2569a3-e246-4f9b-93ce-40478817aaeb)